rollup/plugins/bundlePlugins.js

```diff
@@ -194,11 +194,15 @@ export function makeExcludeBlockPlugin(type) {
   );
 
   const browserIndexFilePath = path.resolve(__dirname, '../../packages/browser/src/index.ts');
+  const browserIntegrationsIndexFilePath = path.resolve(__dirname, '../../packages/browser/src/integrations/index.ts');
```

Comment:
l: Would you mind updating the JSDoc of this function? I think it's pretty outdated by now.

Reply:
actually got rid of this now!